cli/src/util/slice/slice_test.go

```diff
@@ -0,0 +1,54 @@
+package slice
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_SliceContains(t *testing.T) {
+	type testCase struct {
+		slice      interface{}
+		typeOf     string
+		matchValue interface{}
+		wantMatch  bool
+	}
+
+	testCases := []testCase{
```

Comment:
missing a test case for empty slice ?

Reply:
Done in [1874046](https://github.com/openhie/package-starter-kit/pull/36/commits/187404627b4d58a4b2259efa6385229c5a502e28)